lib/ui/views/dashboard_view.dart

```diff
@@ -0,0 +1,154 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+import 'package:stacked/stacked.dart';
+import 'package:table_calendar/table_calendar.dart';
+
+// VIEWMODEL
+import 'package:notredame/core/viewmodels/dashboard_viewmodel.dart';
+
+// WIDGET
+import 'package:notredame/ui/widgets/base_scaffold.dart';
+import 'package:flutter/src/widgets/dismissible.dart';
+
+// OTHER
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:notredame/ui/utils/app_theme.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class DashboardView extends StatefulWidget {
+  @visibleForTesting
+  const DashboardView({Key key}) : super(key: key);
+
+  @override
+  _DashboardViewState createState() => _DashboardViewState();
+}
+
+class _DashboardViewState extends State<DashboardView>
+    with TickerProviderStateMixin {
+  static const Color _selectedColor = AppTheme.etsLightRed;
+
+  static const Color _defaultColor = Color(0xff76859B);
+
+  final DateFormat _dateFormat = DateFormat.MMMMEEEEd();
+
+  CalendarController _calendarController;
+
+  AnimationController _animationController;
+
+  bool isAboutUsVisible = true;
+
+  @override
+  void initState() {
+    super.initState();
+    _calendarController = CalendarController();
+    _animationController = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 400),
+    );
+
+    _animationController.forward();
+  }
+
+  @override
+  void dispose() {
+    _animationController.dispose();
+    _calendarController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<DashboardViewModel>.reactive(
+        viewModelBuilder: () => DashboardViewModel(
+          intl: AppIntl.of(context),
+        ),
+        builder: (context, model, child) => BaseScaffold(
+          isInteractionLimitedWhileLoading: false,
+          appBar: AppBar(
+            title: Text(AppIntl.of(context).title_dashboard),
+            centerTitle: false,
+            automaticallyImplyLeading: false,
+            actions: _buildActionButtons(),
+          ),
+          body: Column(mainAxisSize: MainAxisSize.min, children: [
```

Comment:
use a listview instead to prevent this to happen. Maybe a listview.builder could work also
![image](https://user-images.githubusercontent.com/59664238/111038838-f0cc9700-83f8-11eb-8ea0-47b63a4dcc61.png)